functions/snippets/src/main/java/com/example/functions/LogHelloWorld.java

```diff
@@ -32,8 +32,12 @@
   @Override
   public void service(HttpRequest request, HttpResponse response)
       throws IOException {
-    LOGGER.info("I am an info log!");
-    LOGGER.warning("I am a warning log!");
+    System.out.println("I am an info log to stdout!");
+    System.err.println("I am an info log to stderr!");
+
+    // Stackdriver does not detect severity from Cloud Functions log output
+    LOGGER.info("I am an info log to Stackdriver!");
+    LOGGER.warning("I am an info log to Stackdriver!");
```

Comment:
I thought that we did look for WARNING and INFO and other key words to fill stackdriver

Reply:
This is a known problem according to @vinod-at-google (GCF PM)